src/averageAges.js

```diff
@@ -20,6 +20,11 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  return people
+    .filter(e => e.sex === 'm')
+    .filter(e => century !== undefined
+      ? Math.ceil(e.died / 100) === century : true)
+    .map(e => e.died - e.born).reduce((a, b, i) => (a * i + b) / (i + 1));
```

Comment:
names of the variables make no sense, it's better to replace them with something more meaningful. What is 'e' or 'x'?